add_pypi_package.sh

```diff
@@ -72,40 +88,20 @@ generate_pypi_package() {
 }
 
 add_pypi_to_comps() {
-  local comps_packages=$(rpmspec --query --builtrpms --queryformat '%{NAME}\n' $PACKAGE_DIR/*.spec)
-  if [[ $REPO == katello-* ]]; then
-    local comps_file="katello-server"
-  else
-    local comps_file="foreman"
-  fi
-
-  for comps_package in ${comps_packages}; do
-    if [[ $comps_package != *-debuginfo ]] && [[ $comps_package != *-debugsource ]] ; then
-      ${SCRIPT_ROOT}/add_to_comps.rb comps/comps-${comps_file}-${DISTRO}.xml $comps_package
-    fi
-  done
-  ${SCRIPT_ROOT}/comps_doc.sh
+  "${SCRIPT_ROOT}/add_spec_to_comps" $PACKAGE_DIR/*.spec
   git add comps/
 }
 
-add_pypi_to_manifest() {
-	if [[ $REPO == "foreman-el8" ]] ; then
-		local section="foreman_core_packages"
-	elif [[ $REPO == "foreman-plugins-el8" ]] ; then
-		local section="foreman_plugin_packages"
-	elif [[ $REPO == "katello-el8" ]] ; then
-		local section="katello_packages"
-	else
-		# TODO: client packages
-		local section=""
-	fi
+add_to_package_manifest() {
+  local package="${PACKAGE_NAME}"
+  local section="${MANIFEST_SECTION}"
```

Comment:
We have a weird mix of two spaces and I think 8 spaces throughout this file, this method now highlights that.

Reply:
Fixed